hardware/Rtl433.cpp

```diff
@@ -265,6 +268,24 @@ void CRtl433::Do_Work()
 				}
 				catch (boost::bad_lexical_cast e) {
 				}
+				try {
+					if (!data["depth_cm"].empty())
+					{
+						depth_cm = boost::lexical_cast<float>(data["depth_cm"]);
```

Comment:
could you remove the boost dependencies... you can use atof for instance and if needed with a standard c++ cast